lib/stylesheet/importer.rb

```diff
@@ -253,7 +253,8 @@ def match_theme_import(path, parent_path)
     end
 
     def category_css(category)
-      "body.category-#{category.slug}, body.category-#{category.full_slug} { background-image: url(#{upload_cdn_path(category.uploaded_background.url)}) }\n"
+      full_slug = category.full_slug.split("-")[0..-2].join("-")
```

Comment:
What not use `category.slug_path`?

Does it work with sub-sub-categories?

Reply:
I wanted at first, but `slug_path` is not cached, but `full_slug` is (because `url` is).

Yes, it works with sub-sub-categories. 😊